lib/services/location_service.dart

```diff
@@ -11,7 +11,7 @@ class LocationService {
       _geolocator.checkPermission();
 
   // get users currentLocation
-  Future<Position> getUsersCurrentLocation() =>
+  Future<Position> get getUsersCurrentLocation =>
```

Comment:
just a wee update

Reply:
I think standard practice for a getter is to not have 'get' in the name.  I believe getters are conceptually the same as accessing a member and so should just be the member name.

Reply:
sweet thank you!